clusterloader2/pkg/util/util.go

```diff
@@ -137,6 +147,25 @@ func getDuration(dict map[string]interface{}, key string, defaultValue time.Dura
 	return duration, nil
 }
 
+func getBool(dict map[string]interface{}, key string, defaultValue bool, defaultError error) (bool, error) {
+	value, exists := dict[key]
+	if !exists {
+		return defaultValue, defaultError
```

Comment:
Returning "defaultValue, defaultError" here (and everywhere else in this file) looks really strange here.

Instead you should have a dedicated "notFoundError" and return
"zeroValueForThatType", errNotFound

Reply:
OK. I've changed the functions co they return zero-value every time.